docs/2.4.3.md

```diff
@@ -0,0 +1,41 @@
+# ProxySQL 2.4.3 and ProxySQL admin tools (2022-08-31)
+
+[ProxySQL](https://proxysql.com/) is a high-performance proxy for MySQL and MySQL-compatible database servers such as Percona Server for MySQL and MariaDB. It acts as an intermediary for client requests seeking resources from the database. ProxySQL was created for the database administrator as a solution for complex replication topology issues. 
+
+You can [download the original ProxySQL from GitHub](<https://github.com/sysown/proxysql/releases>). The [ProxySQL documentation](https://proxysql.com/documentation/) provides information on installing and running ProxySQL. 
+
+The [ProxySQL Admin (proxysql-admin)](https://docs.percona.com/proxysql/v2-config.html) tool simplifies the configuration of Percona XtraDB Cluster nodes with ProxySQL. [ProxySQL Admin 1.x.x](https://docs.percona.com/proxysql/proxysql-v1.html) requires custom `Bash` scripts to track the status of a Percona XtraDB Cluster. [ProxySQL Admin 2.x.x](https://docs.percona.com/proxysql/psql-functions.html) supports Percona XtraDB Cluster without custom scripts.
+
+The [Percona Scheduler Admin (percona-scheduler-admin)](https://docs.percona.com/proxysql/psa-scheduler.html) tool can automatically perform a failover due to node failures, service degradation, or maintenance.
+
+!!! note
+
+    Percona Scheduler admin has different features and capabilities than  ProxySQL admin. Do not use the options from one tool in the other tool. Mixing the options may cause unintended results.
+
+## Release Highlights
+
+This release includes all features and bug fixes available in ProxySQL 2.4.3. The improvements and bug fixes introduced by *ProxySQL* and included in this release are:
+
+* Updated to OpenSSL 3.0.5
+* Added an error message, `PMC-10003` about the invalid use of `mysql-init-connect`
+* Fixed the processing of escaped string delimiters for digests
+
+For more information on these items and other improvements and bug fixes, see the *ProxySQL* [2.4.3 release notes](https://github.com/sysown/proxysql/releases/tag/v2.4.3).
+
+*ProxySQL Admin* and *Percona Scheduler Admin* have not changed since the previous release.
```

Comment:
We fixed the following two bugs for 2.4.3 release.

1. https://jira.percona.com/browse/PSQLADM-402 - Positional Parameters in proxysql-status was not working in previous releases. It has been fixed now.
2. https://jira.percona.com/browse/PSQLADM-419 - Addresses CVE-2021-38561 in pxc_scheduler_handler.

Reply:
Added.